medicines/pars-upload/Makefile

```diff
@@ -16,3 +16,15 @@ set-env: ## Takes your current .env file and replaces the keyvault value with th
 	--vault-name mhra-dev \
 	--name pars-env \
 	--file .env
+
+.PHONY: e2e
+e2e: ## Runs end-to-end tests, including dev and mock servers, and cleans up afterwards.
+	yarn mock & wait-on http://localhost:8000; \
+	mock_server_pid = $$!; \
+	NEXT_PUBLIC_DISABLE_AUTH=true yarn dev & wait-on http://localhost:3000; \
+	dev_server_pid=$$!; \
+	test_exit_code=$$?; \
```

Comment:
I think this line should go after `yarn run cypress run` to pick up the exit code for the cypress test run, I think currently it's picking up the exit code of `wait-on http://localhost:3000` above?

Reply:
ah yes! I've reverted back to `yarn test-e2e` being the source of truth, with this `make e2e` being a pointer to it, so that should be fixed now